tools/downloader/pytorch_to_onnx.py

```diff
@@ -65,6 +81,7 @@ def parse_args():
                         help='Comma-separated names of the output layers')
     parser.add_argument('--model-param', type=model_parameter, default=[], action='append',
                         help='Pair "name"="value" of model constructor parameter')
+    parser.add_argument('--inputs-dtype', type=str, required=False, choices=INPUT_DTYPE_TO_TORCH)
```

Comment:
Please, add help message and update script's readme file

Reply:
there is no internal MD scripts readme (only user part - downloader, converter, quantizer are provided in readme),
help message added